modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalStackImageTest.java

```diff
@@ -0,0 +1,50 @@
+package org.testcontainers.containers.localstack;
+
+import org.junit.*;
+import org.junit.rules.TemporaryFolder;
+import org.testcontainers.utility.TestcontainersConfiguration;
+
+import java.io.File;
+import java.io.IOException;
+
+public class LocalStackImageTest {
+
+    @Rule
+    public TemporaryFolder folder = new TemporaryFolder();
+
+    @Before
+    public void setupConfig() throws IOException {
+        File tempDir = folder.newFolder();
```

Comment:
please use system properties instead

Reply:
I assume you were thinking of the other project here as well?